src/OpenDiffix.Core/ParserTypes.fs

```diff
@@ -39,6 +44,7 @@ and Expression =
   | Join of joinType: JoinType * left: Expression * right: Expression * on: Expression
   | SubQuery of subQuery: SelectQuery * alias: string
   | Function of functionName: string * Expression list
+  | OrderSpec of expr: Expression * direction: Expression option * nullsBehavior: Expression option
```

Comment:
If default behavior is handled during parsing, there is no more need for the `option` attribute here.